pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2024,14 +2071,21 @@ type RestoreSpec struct {
 	Env []corev1.EnvVar `json:"env,omitempty"`
 	// To is the tidb cluster that needs to restore.
 	To *TiDBAccessConfig `json:"to,omitempty"`
-	// Type is the backup type for tidb cluster.
+	// Type is the backup type for tidb cluster and only used when Mode = snapshot, such as full, db, table.
 	Type BackupType `json:"backupType,omitempty"`
+	// Mode is the restore mode. such as snapshot or pitr.
+	// +kubebuilder:default=snapshot
+	Mode RestoreMode `json:"restoreMode,omitempty"`
+	// RestoredTs is the pitr restored ts.
+	RestoredTs string `json:"restoredTs,omitempty"`
```

Comment:
There is another parameter called `start-ts`, which means when restore from. If we only develop PiTR function, we needn't add this parameter. But if we also support to restore log backup without full backup, we need add this parameter.

Reply:
ok, I haven't tried it either.  i will add `start-ts`.

Reply:
add at [this](https://github.com/pingcap/tidb-operator/pull/4648/commits/89b00ea60b80fd3395eec4c75750f1ab1e31baf6)